src/platform/Kconfig

```diff
@@ -136,9 +140,14 @@ config IMX8
 
 endchoice
 
+config INTEL
```

Comment:
Can you add some help text here? It will help people understand later what is this all about.

Reply:
I didn't know what to write there, because there is not so much to write, but I added something.